DEPS

```diff
@@ -713,6 +713,14 @@ hooks = [
       'src/build/win/generate_winrt_headers.py',
     ]
   },
+  {
+    'name': 'Download prebuilt Dart SDK',
```

Comment:
Should we guard this with some kind of variable? Will downloading this always be necessary - in particular, will there be any builders that still use --full-dart-sdk and so should skip this?

Reply:
Good idea. The API in the recipes appears to use env vars, so I added a guard on one in the python script. Next step is to enable in recipes and see how much things get better there.